Translation corrections (#399)
07th-mod/higurashi-console-arcs: Update/omot_009_tobadend.txt

//r…俺たちはその後、どうでもいい世間話をしてから、店を出た。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…俺たちはその後、どうでもいい世間話をしてから、店を出た。",

> 			NULL, "...Afterwards, we made our way out of the store after making some chitchat.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Afterwards, we made some chitchat and then left the store.", Line_Normal);
```
